jitsi-utils/src/test/java/org/jitsi/utils/BasicByteArrayBuffer.java

```diff
@@ -1,5 +1,10 @@
 package org.jitsi.utils;
 
+import edu.umd.cs.findbugs.annotations.*;
+
+@SuppressFBWarnings(
+    value = "EI_EXPOSE_REP",
+    justification = "It is the nature of this class to expose 'buffer.'")
```

Comment:
Is this firing because we have a private member that we return via the getter?

Reply:
No, it's the same one we discussed in jitsi-rtp: the class has a `private` field of an array type, which is also exposed publically. Here it it exposed via `getBuffer`, in rtp it was indirectly exposed because it was set externally.